src/parsec_client/operations/list_authenticators.md

```diff
@@ -0,0 +1,43 @@
+# ListAuthenticators
+
+Gets a list of available Parsec authenticators available for use. Opcode: 14 (`0x0014`)
```

Comment:
Seems like you repeat "available" twice 😛 Also, I think the list returned by this operation is ordered? Last thing, it might be nice to indicate "available for use at the Listener endpoint" at the end to inform it is only for the listener endpoint used to perform the request.

Reply:
Good spot on the double-use of 'available' -- thanks for that!

And yes, this operation is ordered, I just wasn't sure if we had a specific ordering in mind. I agree with your other comment, though -- we should order the authenticators by priority (i.e. the primary authenticator comes first).

Will amend!